committeeoversightapp/templates/partials/landing_page_table.html

```diff
@@ -9,7 +9,7 @@ <h4 class="text-center mt-4">{{title}}</h4>
       <thead>
         <tr class="landing-row">
           <th class="align-top">Committee</th>
-          <th class="align-top">Current Congress (projected){% if current_congress.footnote %}*{% endif %}</th>
+          <th class="align-top">Current Congress {% if current_congress.is_current %} (projected){% endif %}{% if current_congress.footnote %}*{% endif %}</th>
```

Comment:
I think "Current Congress" is a little misleading here, as the 116th Congress is no longer current.

On the other hand, Jamie's ask was:

> On landing page, change header in second column from “Current Congress (projected)” to “Final Grade” until grades change to 117th Congress on April 1, 2021.

This also seems underspecified. 

What if we showed the display name of `current_congress`, e.g., `116th Congress`, instead of "Current Congress"?

Reply:
thats a good point. i'll make that change